### PR TITLE
[20.09] Fix permission in user dir library import

### LINKED ISSUE #10982: Option to add data sets "from user directory" in 20.09
Seems that on 20.09 the the menu in the shared libraries view does not contain: 

```
Add Data Sets -> From User Directories
```

Was working before and as far as I can see my configuration did not change in this respect.

### PR DESCRIPTION
**Pull request by OlegZharkov**
fixes https://github.com/galaxyproject/galaxy/issues/10982
Also hides import modal on `404001` error

### CODE REVIEW

File: lib/galaxy/managers/configuration.py [hunk 1]
```diff
@@ -108,6 +108,7 @@ def _use_config(config, key, **context):
             'file_sources_configured'           : lambda config, key, **context: self.app.file_sources.custom_sources_configured,
             'upload_from_form_button'           : _use_config,
             'release_doc_base_url'              : _use_config,
+            'user_library_import_dir_available' : lambda config, key, **context: True if config.get('user_library_import_dir') else False,
```
Comment (mvdbeek) [inline lib/galaxy/managers/configuration.py]:
```suggestion
            'user_library_import_dir_available' : lambda config, key, **context: bool(config.get('user_library_import_dir')),
```
that's a little simpler and what we're doing for `server_mail_configured ` above.

Comment (OlegZharkov) [reply lib/galaxy/managers/configuration.py]: thanks

**Comment by bernt-matthias** [comment]
Wonderful. Is there a way to test this functionality?

**Comment by OlegZharkov** [comment]
> Wonderful. Is there a way to test this functionality?

You're definitively welcome to `gh pr checkout 11137` and try it out locally

**Comment by bernt-matthias** [comment]
I meant something in the CI test suite.

Currently it's difficult for me to test this because I can't restart my instance.. Hope that our admins fix this soon..

**Review by dannon — APPROVED** [review]

**Comment by OlegZharkov** [comment]
thanks a lot for your help on this @dannon

**Comment by OlegZharkov** [comment]
> I meant something in the CI test suite.
> 
> Currently it's difficult for me to test this because I can't restart my instance.. Hope that our admins fix this soon..

here https://github.com/galaxyproject/galaxy/pull/11203

**Comment by bernt-matthias** [comment]
Btw. I was able to test this on my instance. Works like a charm. Thanks a lot.

**Pull request merged**